data/quickstarts/gpu-enabled-notebook-quickstart.yaml

```diff
@@ -0,0 +1,42 @@
+kind: ConsoleQuickStart
+metadata:
+  name: gpu-enabled-notebook-quickstart
+  annotations:
+    opendatahub.io/categories: 'Getting started,GPUs,Python'
+spec:
+  displayName: Creating a Machine Learning Model using the NVIDIA GPU Add-on.
+  appName: nvidia
+  durationMinutes: 5
+  icon: 'images/nvidia.svg'
+  description: Install GPU add-on
+  introduction: |-
+    ### This quick start shows you how to run a JupyterHub Notebook that uses the available GPUs.
+  tasks:
+    - title: Import a JupyterHub Notebook into JupyterLab
+      description: |-
+        1. In the top navigation panel click **Git**
+        2. Click **Clone a repository**.
+        3. Enter URL of the repository to be cloned : **https://github.com/rh-aiservices-bu/getting-started-with-gpus** and click **Clone**.
```

Comment:
Shouldn't the first step of this quickstart tell the user to spawn a notebook from the spawner with at least 1 GPU attached?

Reply:
This quickstart is supposed to be run after the "Installing and verifying the nvidia gpu add-on", which contains steps to spawn a notebook with at least 1 GPU attached. I can mention in the introduction that the other quickstart is a pre-requisite to this one

Reply:
I think that's a good compromise

Reply:
@VedantMahabaleshwarkar there should be a way to set the "pre-requisites" of a quick start. It will show as a sub-section of the introduction. You can look at `Creating a Jupyter notebook` for an example